cmd/temporalite/main.go

```diff
@@ -155,6 +157,11 @@ func buildCLI() *cli.App {
 					return cli.Exit(fmt.Sprintf("ERROR: only one of %q or %q flags may be passed at a time", ephemeralFlag, dbPathFlag), 1)
 				}
 
+				err := ensureDirExists(filepath.Dir(c.String(dbPathFlag)))
```

Comment:
```suggestion
				err := os.MkdirAll(filepath.Dir(c.String(dbPathFlag)), os.ModePerm)
```

You basically just rewrote what `MkdirAll` already does in your function

Reply:
LOL, you are right. I updated the PR.